MarkBind/vue-strap
# Remove unused documentation

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [X] Other, please explain: Removing unused documentation


Part of MarkBind/markbind#289.



**What is the rationale for this request?**
The documentation website is no longer in use. 
* These documentation have been migrated to our MarkBind's user guide (MarkBind/markbind#274).
* It has always been broken since the migration to Vue 2.0 (#50), so there is no point in keeping them, because the deployed website has never been really updated since then.

**What changes did you make? (Give an overview)**
1. Delete all the files!
1. Removed the npm command for building docs.

**Provide some example code that this change will affect:**
NA

**Is there anything you'd like reviewers to focus on?**
NA

**Testing instructions:**
Ensure that building the actual library still works (i.e. `npm run build`), as it should not be affected by the removal of documentation building.
